public/pages/utils/anomalyResultUtils.ts

```diff
@@ -77,19 +95,22 @@ export const getLiveAnomalyResults = (
 export const buildParamsForGetAnomalyResultsWithDateRange = (
   startTime: number,
   endTime: number,
-  anomalyOnly: boolean = false
+  anomalyOnly: boolean = false,
+  entity: Entity | undefined = undefined
 ) => {
   return {
     from: 0,
     size: MAX_ANOMALIES,
     sortDirection: SORT_DIRECTION.DESC,
     sortField: AD_DOC_FIELDS.DATA_START_TIME,
-    dateRangeFilter: {
+    dateRangeFilter: JSON.stringify({
```

Comment:
This can be parsed and converted in [here](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/master/server/routes/ad.ts#L725-L727). Actually this change fixes bug which is not found before: dateRangeFilter is not parsed and passed to ES.